tests/gis_tests/gdal_tests/test_geom.py

```diff
@@ -320,6 +320,11 @@ def test_srs(self):
                     self.assertEqual('WGS 72', ring.srs.name)
                     self.assertEqual(4322, ring.srid)
 
+            # test assigning of SRS and SRID to None
```

Comment:
Minor nit, but I prefer stating the expected behavior like this: `# SRS and SRID may be assigned to None.` since all tests are obviously for testing things.